src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java

```diff
@@ -0,0 +1,87 @@
+package org.broadinstitute.dsde.consent.ontology.resources;
+
+import org.apache.commons.lang3.StringUtils;
+import org.parboiled.common.FileUtils;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriInfo;
+
+@Path("/swagger")
+public class SwaggerResource {
+
+    private final static String swaggerResource = "META-INF/resources/webjars/swagger-ui/2.2.8/";
```

Comment:
what about if we move this path to config file, because if we update swagger version we will need to update this class

Reply:
I like that idea, but would like to hold off on that until some other PRs are merged. Will follow up with a story for it.

Reply:
Also, would like to tackle this for consent too.